packages/frames.js/src/types.ts

```diff
@@ -91,7 +91,7 @@ export type FrameButtonPost = {
 };
 export type FrameButtonPostRedirect = FrameButtonPost;
 
-export type FrameButton =
+export type FrameButtonType =
```

Comment:
This would be a breaking change, why is it necessary to change the type name?

Reply:
[done](https://github.com/framesjs/frames.js/pull/77/commits/818ffcc14e44c6a50f5c096a81588ebde08986e9)